vignettes/enrich.Rmd

```diff
@@ -0,0 +1,79 @@
+
+---
+title: "Enrichment analysis workflow"
+author: "Ru Feng"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{intro-to-pecotmr}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+This vignette demonstrates the enrichment analysis part of `pecotmr` package, which largelly follows from fastenloc https://github.com/xqwen/fastenloc but uses `susieR` fitted objects as input to estimate prior for use with `coloc` package (coloc v5, aka SuSiE-coloc).
+The main differences are:
+1) now enrichment is based on all QTL variants whether or not they are inside signal clusters;
+2) Causal QTL are sampled from SuSiE single effects, not signal clusters;
+3) Allow a variant to be QTL for not only multiple conditions (eg cell types) but also multiple regions (eg genes).
+
+
+## Set up environment and Load data
+
+Load the `pecotmr` package for this enrichment analysis, abd load the susieR package for data checking and visualization
+```{r load-pkgs}
+library(pecotmr)
+library(susieR)
+```
+
+Load fine mapping results. 
+```{r}
+gwas_path <- "gwas_toy.rds"
+qtl_path <- "QTL_toy.rds"
+gwas <- readRDS(gwas_path)
+qtl <- readRDS(qtl_path)
+```
+
+## check data structure
```

Comment:
Be consistent with capitalization -- "C"heck not "c"heck. Also "check data structure" does not sound as good as something like Input data requirements"